hedera-node/configuration/dev/bootstrap.properties

```diff
@@ -1,4 +1,4 @@
-bootstrap.throttleDefsJson.resource=throttles-dev.json
+bootstrap.throttleDefsJson.resource=genesis/throttles-dev.json
```

Comment:
Is this file still used? Modular code should be using `application.properties`

Reply:
So, we copy the bootstrap properties from this repo into genesis.properties. We have to get this file renamed to genesis.properties in the repo I think. Until then we just copy it to the new name.

Reply:
Should throttle defs be in genesis.properties or application.properties?

Reply:
Would you be okay with merging as is and improve with a followup based on the outcome of this discussion?

Reply:
Yes @ElijahLynn . Just was curious why this is still used. We can remove usage of this file later.